src/Data/List/Ext.agda

```diff
@@ -4,9 +4,63 @@ module Data.List.Ext where
 open import Agda.Builtin.List using (List; []; _∷_)
 open import Agda.Builtin.Maybe using (Maybe; nothing; just)
 open import Agda.Builtin.Nat using (Nat; zero; suc)
+open import Data.List using (List; [_]; _++_; map)
+open import Data.List.Membership.Propositional using (_∈_)
+open import Data.List.Relation.Unary.All using (All)
+open import Function using (_∘_)
+open import Level using (Level)
+
+private variable
+  ℓ : Level
+  A : Set ℓ
 
 -- Looking up an index into the list; fails when out-of-bounds.
 _⁉_ : ∀ {a}{A : Set a} → List A → Nat → Maybe A
 []       ⁉ _     = nothing
 (x ∷ _)  ⁉ zero  = just x
 (_ ∷ xs) ⁉ suc n = xs ⁉ n
+
```

Comment:
These are supporting functions, some of which will likely be needed to fill the two holes that remain in `Ledger.Gov`.